engine/src/mbliphoneextra.mm

```diff
@@ -1076,6 +1076,10 @@ static Exec_stat MCHandleClearTouches(void *context, MCParameter *p_parameters)
 	MCscreen -> wait(1/25.0, False, False);
 	static_cast<MCScreenDC *>(MCscreen) -> clear_touches();
 	MCEventQueueClearTouches();
+    
+    // PM-2015-03-16: [[ Bug 14333 ]] Make sure the object that triggered a mouse down msg is not focused, as this stops later mouse downs from working
+    MCtargetptr -> munfocus();
```

Comment:
Whilst unlikely, it is possible for MCtargetptr to be NIL in certain cases (other places in the code which aren't event handlers do guard it) - therefore make sure you only do this `if (MCtargetptr != nil)` just to be on the safe side.